### ISSUE TITLE
increase temporal resolution of Sentinel 3 collections

### ISSUE THREAD

**Issue by bossie** [Open-EO/openeo-geopyspark-driver#942]
https://forum.dataspace.copernicus.eu/t/issues-with-sentinel-3-lst-and-view-zenith-angles/1371

A temporal resolution of an hour does not suffice; try minutes instead?

**Comment by jdries** [comment]
Agreed. @JeroenVerstraelen  this is relatively easy to try on staging, so would like to plan this for next sprint at latest.
